internal/generator/fluentd/output/http/http_test.go

```diff
@@ -319,6 +320,90 @@ var _ = Describe("Generate fluentd config", func() {
 	</buffer>
   </match>
 </label>
+`,
+		}),
+		Entry("with TLS config but not secure URL", helpers.ConfGenerateTest{
+			CLFSpec: logging.ClusterLogForwarderSpec{
+				Outputs: []logging.OutputSpec{
+					{
+						Type: logging.OutputTypeHttp,
+						Name: "http-receiver",
+						URL:  "http://my-logstore.com/logs/app-logs",
```

Comment:
IMO this is a user error because non secure protocol + TLS makes no sense.  We should reject this as a validation error before even attempting to create a configuration